nion/swift/ExportDialog.py

```diff
@@ -101,6 +101,28 @@ def on_writer_changed(self, widget: Declarative.UIWidget, current_index: int) ->
         writer = self.__writers[current_index]
         self.viewmodel.writer.value = writer
 
+    def on_prefix_changed(self, widget: Declarative.UIWidget, text: str) -> None:
+        self.validate_options()
+
+    def validate_options(self) -> bool:
+        invalid_reason: str | None = None
+
+        prefix_text = self.viewmodel.prefix.value or ""
+        if prefix_text != Utility.simplify_filename(prefix_text) or "\\" in prefix_text or "/" in prefix_text:
```

Comment:
Doesn't `simplify_filename` already catch forward and backslashes? If not, why doesn't it?

Reply:
Ping